docs/src/content/docs/references/communication.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Agent Communication
+description: References - Agent Communication
+---
+
+Eidolon simplifies agent-to-agent communication with a built-in mechanism enabling seamless interaction between agents. In the example below we will reuse the `hello_world` agent from our quickstart guide, and create a second `qa` agent who will interface with the hello world agent.
+
+_hello_world_agent.yaml_
+```yaml
+apiVersion: eidolon/v1
+kind: Agent
+metadata:
+  name: hello_world
+
+spec:
+  implementation: GenericAgent
```

Comment:
We want to deprecate the GenericAgent. Can you move this example to the "SimpleAgent" (used by default) or make a separate issue to make the changes?

Reply:
Issue opened: [#174](https://github.com/eidolon-ai/eidolon/issues/174)